### PR TITLE
Create PR and issue templates

### PR DESCRIPTION
**Pull request by jcpitre**
Added issue and PR templates.
Mostly copied from gtfs-validator, with some adjustments.
Closes https://github.com/MobilityData/mobility-feed-api/issues/27

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+newPRWelcomeComment: >
+  Thanks for opening this pull request! You're awesome.
+  We use [semantic commit messages](https://github.com/electron/electron/blob/main/docs/development/pull-requests.md#commit-message-guidelines) to streamline the release process. Before your pull request can be merged, you should **update your pull request title** to start with a semantic prefix.
+  Examples of titles with semantic prefixes:
+    - `fix: Bug with ssl network connections + Java module permissions.`
+    - `feat: Initial support for multiple @PrimaryKey annotations`.
+    - `docs: update RELEASE.md with new process`
+  To get this PR to the finish line, please do the following:
+    - Read our [Contribution Guidelines](https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md)
+    - Follow [Google Java style](https://google.github.io/styleguide/javaguide.html) coding standards
```
Comment (davidgamez) [inline .github/config.yml:23]: Let's remove these lines until we have a similar process for the python code.

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue in this project! If you haven't already, you can [join our slack](https://share.mobilitydata.org/slack) and join the #gtfs-validators channel to meet our awesome community. Come say hi :wave:!
```
Comment (davidgamez) [inline .github/config.yml:7]: Replace slack channel,
```suggestion
  Thanks for opening your first issue in this project! If you haven't already, you can [join our slack](https://share.mobilitydata.org/slack) and join the #mobility-database channel to meet our awesome community. Come say hi :wave:!
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    
+- type: input
+  attributes:
+    label: Operating system
+    description: |
+      Which Operating System are you using?
+    placeholder: >
+      Example: MacOS, Apple M1
+  validations:
+    required: true
```
[the remark this answers is not in the document]
Comment (jcpitre) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Removed

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    
+- type: input
```
[the remark this answers is not in the document]
Comment (jcpitre) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Removed

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+Summarize the changes in the pull request including how it relates to any issues (include the #number, or link them).
+
+**Expected behavior:** 
+
+Explain and/or show screenshots for how you expect the pull request to work in your testing (in case other devices exhibit different behavior).
+
+
+Please make sure these boxes are checked before submitting your pull request - thanks!
+
+- [ ] Run the unit tests with `gradle test` to make sure you didn't break anything
```
[the remark this answers is not in the document]
Comment (jcpitre) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done

File: .github/config.yml (hunk 2 above)
Comment (jcpitre) [reply .github/config.yml]: Done

**Review by davidgamez — APPROVED** [review]

File: .github/config.yml (hunk 1 above)
Comment (jcpitre) [reply .github/config.yml]: Done

**Pull request merged**